Content.Server/Remotes/DoorRemoteSystem.cs

```diff
@@ -85,7 +92,7 @@ private void OnBeforeInteract(EntityUid uid, DoorRemoteComponent component, Befo
             switch (component.Mode)
             {
                 case OperatingMode.OpenClose:
-                    if (_doorSystem.TryToggleDoor(args.Target.Value, doorComp, args.Used))
+                    if (_doorSystem.TryToggleDoor(args.Target.Value, doorComp, args.User))
```

Comment:
This is probably the most substantial change in the PR, and yeah it looks minor in the diff. 

Rather than looking for access only in the remote (args.Used) it now looks for it in the player (args.User). So it also then looks for it within any item the player is carrying, such as an ID card or the remote itself.

Reply:
I added a relevant comment since it's an easy detail to miss

Reply:
okay makes sense